src/watch/hearRingiOSWatchOS/hearRingiOSWatchOS WatchKit Extension/Info.plist

```diff
@@ -2,11 +2,7 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-    <key>NSMicrophoneUsageDescription</key>
```

Comment:
is this supposed to be gone? or is it the same deal as my stuff just disappearing